phinx.json

```diff
@@ -0,0 +1,36 @@
+{
+  "paths": {
+    "migrations": "%%PHINX_CONFIG_DIR%%/migrations"
```

Comment:
On second thought, users wishing to use yaml will likely have the extension anyway. Ok. I'll go ahead with this.